modules/op-release-notes-1-10.adoc

```diff
@@ -211,4 +211,16 @@ With this update, {pipelines-title} General Availability (GA) 1.10.4 is availabl
 
 * Before this update, the `pipelinerun.timeouts` field was reset to the `timeouts.pipeline` value, ignoring the `timeouts.tasks` and `timeouts.finally` values. This update fixes the issue and sets the correct default timeout value for a `PipelineRun` resource.
 
-* Before this update, the controller logs contained unnecessary data. This update fixes the issue.
\ No newline at end of file
+* Before this update, the controller logs contained unnecessary data. This update fixes the issue.
+
+[id="release-notes-1-10-5_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.5
+
+With this update, {pipelines-title} General Availability (GA) 1.10.5 is available on {product-title} 4.10 in addition to 4.11, 4.12, and 4.13.
```

Comment:
we need to add that 1.10.5 will be only available in the `pipelines-1.10` channel on OCP 4.10, 4.11, 4.12, 4.13 but will not be available in the `latest` channel for any OCP version including OCP 4.11